aries-site/src/layouts/content/Example/Container.js

```diff
@@ -27,31 +28,71 @@ export const Container = ({
   else if (!plain) height = { min: 'medium' };
   else height = undefined;
 
+  let bestPractice;
+  let background;
+  let label;
+  if (bestPracticeProp) {
+    if (bestPracticeProp.type === 'do') {
+      background = 'brand';
```

Comment:
Resolves to same color value, but semantically brand doesn't feel as appropriate here.

```suggestion
      background = 'green!';
```